pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -167,3 +170,23 @@ func (pl *TaintToleration) ScoreExtensions() framework.ScoreExtensions {
 func New(_ context.Context, _ runtime.Object, h framework.Handle) (framework.Plugin, error) {
 	return &TaintToleration{handle: h}, nil
 }
+
+// isSchedulableAfterPodChange is invoked whenever a pod changed. It checks whether
+// that change made a previously unschedulable pod schedulable.
```

Comment:
```suggestion
// that change made a previously unschedulable pod schedulable.
// When an unscheduled Pod, which was rejected by TaintToleration, is updated to newly have toleration, 
it may make the Pod schedulable.
```